testing/src/test/java/org/motechproject/nms/testing/it/imi/TargetFileServiceBundleIT.java

```diff
@@ -199,13 +188,42 @@ public void restoreSettings() {
 
     @Test
     public void testTargetFileGeneration() throws NoSuchAlgorithmException, IOException {
-        setupDatabase();
+
+        Subscriber subscriber1 = new Subscriber(1111111111L, hindi, aa);
+        subscriber1.setLastMenstrualPeriod(DateTime.now().minusDays(90)); // startDate will be today
+        subscriberDataService.create(subscriber1);
+        subscriptionService.createSubscription(1111111111L, hindi, pregnancyPack, SubscriptionOrigin.MCTS_IMPORT);
+
+
+        // Should not be picked up because it's been deactivated
+        Subscriber subscriber2 = new Subscriber(2222222222L, urdu, bb);
+        subscriber2.setLastMenstrualPeriod(DateTime.now().minusDays(90)); // startDate will be today
+        subscriberDataService.create(subscriber2);
+        Subscription subscription2 = subscriptionService.createSubscription(2222222222L, urdu, pregnancyPack,
+                SubscriptionOrigin.MCTS_IMPORT);
+        subscriptionService.deactivateSubscription(subscription2, DeactivationReason.CHILD_DEATH);
+
+        //Should not be picked up because it's not for today
+        Subscriber subscriber3 = new Subscriber(6666666666L, urdu, bb);
+        subscriber3.setDateOfBirth(DateTime.now()); // startDate is DOB + 1 for child packs, so setting the DOB
```

Comment:
maybe an edge case - but this test could fall on the day boundary and cause failures as we roll on to dob+1 ? Can we set this to DateTime.now() + 1?